src/pages/ApplyVerify.tsx

```diff
@@ -0,0 +1,110 @@
+import { useState } from 'react';
+
+import BlockButton from '@/components/common/button/BlockButton';
+import LabelButton from '@/components/common/button/LabelButton';
+import Icon from '@/components/common/icon/Icon';
+import InputField from '@/components/common/input/InputField';
+import Label from '@/components/common/label/Label';
+import ProgressIndicator from '@/components/common/progress/ProgressIndicator';
+import Title from '@/components/common/title/Title';
+import { APPLY_TITLE } from '@/constants/applyPageData';
+
+function ApplyVerify() {
+  const [isReady, setIsReady] = useState(false);
```

Comment:
(const step과의 연관성)
```suggestion
  const [isStepComplete, setIsStepComplete] = useState(false);
```
도 생각이 나는데, 참고 사항입니다~ 

++) setIsReady 는 #81  와 마찬가지로 사용되지 않는 값이라 lint 에러가 Qodana에서 발생하네요. 이 부분은 개인적으론 오류라고 판단하지 않아도 될 거 같네요

Reply:
넵..! 맞습니다 #81에도 코멘트 달아두었는데 버튼의 활성/비활성 색상 UI를 설정해두기위해서 그리고 다음 작업 때 사용될 것을 예상하기 때문에 넘어가도 괜찮을 것 같아요 
네이밍은 이 페이지에서 보면 확실히 isStepComplete도 좋은 네이밍이라고 생각합니다! 전단계, 다음 단계 페이지에서도 isReady라는 같은 네이밍을 사용하고 있어서 페이지들끼리 통일을 하려고했던 의도였는데 고민이네요!

Reply:
`isStepCompleted` 로 네이밍 수정했습니다! 다른 페이지도 isReady보다 이거 사용하는게 좋을 것 같아요!